letsdrill.rb

```diff
@@ -1,13 +1,37 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  case integer
+  when 90..100
+    return "A"
```

Comment:
No need for explicit returns here.

http://ruby-for-beginners.rubymonstas.org/writing_methods/return_values.html
https://stackoverflow.com/questions/12061485/what-is-returned-in-ruby-if-the-last-statement-evaluated-is-an-if-statement